client/src/modules/payroll/weekend_configuration/configuration.service.js

```diff
@@ -0,0 +1,32 @@
+angular.module('bhima.services')
+  .service('ConfigurationWeekEndService', ConfigurationWeekEndService);
+
+ConfigurationWeekEndService.$inject = ['PrototypeApiService', '$uibModal', '$http', 'util'];
+
+/**
+ * @class ConfigurationWeekEndService
+ * @extends PrototypeApiService
+ *
+ * @description
+ * Encapsulates common requests to the /weekend_config/ URL.
+ */
+function ConfigurationWeekEndService(Api, Modal, $http, util) {
```

Comment:
It doesn't look like Modal is ever used.  We should avoid having unused imports or functions.  The inflate our codebase and confuse people trying to read the code.